samples/UDPEchoServ4.js

```diff
@@ -0,0 +1,14 @@
+var PORT = 33333;
```

Comment:
As discussed on IRC, as I'm contributing from my company, to a product of another company, my interpretations are irrelevant, and I'll just follow existing practice of the codebase. Fixed.